src/Api/Models/Api/Request/SendRequestModel.cs

```diff
@@ -72,7 +72,7 @@ public Send ToSend(Send existingSend, ISendService sendService)
                         new JsonSerializerSettings { NullValueHandling = NullValueHandling.Ignore });
                     break;
                 case SendType.Text:
-                    existingSend.Data = JsonConvert.SerializeObject(new SendTextData(this),
+                    existingSend.Data = JsonConvert.SerializeObject(new SendTextData(Name, Notes, Text.Text, Text.Hidden),
```

Comment:
Change to the `ToSendTextData` pattern, which is more common in the other models

Reply:
Done, but I made it private since it feels a bit internal. I'm not really happy with our current data models for sends or ciphers.

Reply:
me either, but consistency will make it easier to change down the road. Thanks